{'description': {'en': 'User overlay for jtriley'},
 'feed': ['https://cgit.gentoo.org/user/jtriley.git/atom/'],
 'homepage': 'https://cgit.gentoo.org/user/jtriley.git/',
 'name': 'jtriley',
 'owner': [{'email': 'jtriley@MIT.EDU',
            'name': 'Justin Riley',
            'type': 'person'}],
 'quality': 'experimental',
 'source': [{'type': 'git',
             'uri': 'git://anongit.gentoo.org/user/jtriley.git'},
            {'type': 'git',
             'uri': 'https://anongit.gentoo.org/git/user/jtriley.git'},
            {'type': 'git',
             'uri': 'git+ssh://git@git.gentoo.org/user/jtriley.git'}],
 'status': 'unofficial'}
$ pmaint sync jtriley
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
Already up-to-date.
*** syncing jtriley
*** synced jtriley
 * Sync succeeded
$ pmaint regen -t 2 jtriley
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'

!!! ERROR: app-misc/g15daemon-9999 failed.

!!! ERROR: dev-lua/lunatic-python-9999 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:

!!! ERROR: app-misc/g15daemon-1.9.5.3-r8 failed.
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'

!!! ERROR: app-misc/g15daemon-1.9.5.3-r7 failed.
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '178' 'depend'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '203' 'depend'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '206' 'depend'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/jtriley/app-misc/g15daemon/g15daemon-9999.ebuild'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '206' 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/jtriley/dev-lua/lunatic-python/lunatic-python-9999.ebuild'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/jtriley/dev-lua/lunatic-python/lunatic-python-9999.ebuild'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/jtriley/app-misc/g15daemon/g15daemon-9999.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/jtriley/dev-lua/lunatic-python/lunatic-python-9999.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/jtriley/app-misc/g15daemon/g15daemon-1.9.5.3-r8.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/jtriley/app-misc/g15daemon/g15daemon-1.9.5.3-r8.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/jtriley/app-misc/g15daemon/g15daemon-1.9.5.3-r7.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/jtriley/app-misc/g15daemon/g15daemon-9999.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/jtriley/app-misc/g15daemon/g15daemon-1.9.5.3-r7.ebuild'
!!!          g15daemon-9999.ebuild, line  12:  Called inherit 'eutils' 'linux-info' 'perl-module' 'python' 'base' 'subversion' 'autotools'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/jtriley/app-misc/g15daemon/g15daemon-1.9.5.3-r7.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/jtriley/app-misc/g15daemon/g15daemon-1.9.5.3-r8.ebuild'
!!!     lunatic-python-9999.ebuild, line   8:  Called inherit 'git' 'multilib' 'toolchain-funcs' 'flag-o-matic' 'eutils'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'perl-module'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/perl-module.eclass'
!!!    g15daemon-1.9.5.3-r8.ebuild, line  10:  Called inherit 'eutils' 'linux-info' 'perl-module' 'python' 'base'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'perl-module'
!!!    g15daemon-1.9.5.3-r7.ebuild, line  10:  Called inherit 'eutils' 'linux-info' 'perl-module' 'python' 'base'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'perl-module'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/gentoo/eclass/perl-module.eclass'

!!! ERROR: sci-chemistry/vmd-1.9.1-r2 failed.
!!!             perl-module.eclass, line  40:  Called die
!!! The die message:
!!!  EAPI=4 is not supported by perl-module.eclass

!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/perl-module.eclass'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/perl-module.eclass'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/gentoo/eclass/perl-module.eclass'
!!!             perl-module.eclass, line  40:  Called die
!!! The die message:
!!!  EAPI=4 is not supported by perl-module.eclass

!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/gentoo/eclass/perl-module.eclass'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             perl-module.eclass, line  40:  Called die
!!! The die message:
!!!  EAPI=4 is not supported by perl-module.eclass

!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '176' 'depend'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/jtriley/sci-chemistry/vmd/vmd-1.9.1-r2.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/jtriley/sci-chemistry/vmd/vmd-1.9.1-r2.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/jtriley/sci-chemistry/vmd/vmd-1.9.1-r2.ebuild'
!!!            vmd-1.9.1-r2.ebuild, line   9:  Called inherit 'cuda' 'eutils' 'multilib' 'prefix' 'python' 'toolchain-funcs'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cuda'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 161' for eclass cuda

received unknown com: export CATEGORY='dev-python' PR=r0 PVR='0.7.1' PV='0.7.1' EBUILD='/home/mgorny/repos/jtriley/dev-python/scp/scp-0.7.1.ebuild' P='scp-0.7.1' PF='scp-0.7.1' PN=scppath
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: dev-lua/lunatic-python-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sci-chemistry/vmd-1.9.1-r2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: app-misc/g15daemon-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: app-misc/g15daemon-1.9.5.3-r8
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: app-misc/g15daemon-1.9.5.3-r7
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/scp-0.7.1
 * Cache regenerated successfully
